proto/buf.yaml

```diff
@@ -21,4 +22,4 @@ lint:
     - PACKAGE_VERSION_SUFFIX
     - RPC_REQUEST_STANDARD_NAME
   ignore:
-    - proofs.proto # TODO: remove this when ics23 is linted and can be obtained via buf mod
\ No newline at end of file
+    - proofs.proto # TODO: remove this when ics23 is linted and can be obtained via buf mod
```

Comment:
We can remove this now that ics23 proofs.proto has been deleted from our proto directory